src/schema/objects/metadata.yaml

```diff
@@ -1739,6 +1771,76 @@ NegativeContrast:
     usage of a contrast agent in conjunction with a T2\* weighted acquisition
     protocol.
   type: boolean
+NIRSChannelCount:
+  name: NIRSChannelCount
+  description: |
+    Number of NIRS channels.
+  type: integer
+  minimum: 0
+NIRSSourceOptodeCount:
+  name: NIRSSourceOptodeCount
+  description: |
+    Number of fNIRS sources.
+  type: integer
+  minimum: 1
+NIRSDetectorOptodeCount:
+  name: NIRSDetectorOptodeCount
+  description: |
+    Number of fNIRS detectors.
+  type: integer
+  minimum: 1
+NIRSPlacementScheme:
+  name: NIRSPlacementScheme
+  description: |
+    Placement scheme of NIRS optodes.
+    Either the name of a standardized placement system (for example, `"10-20"`)
+    or an array of standardized position names (for example, `["Cz", "Pz"]`).
+    This field should only be used if a cap was not used.
+    If a standard cap was used, then it should be specified in `CapManufacturer`
+    and `CapManufacturersModelName` and this field should be set to `"n/a"`
+  anyOf:
+  - type: string
+  - type: array
+    items:
+      type: string
+NIRSCoordinateSystem:
+  name: NIRSCoordinateSystem
+  description: |
+    Defines the coordinate system in which the optode positions are expressed.
+
+    See
+    [Appendix VIII](/99-appendices/08-coordinate-systems.html)
+    for a list of restricted keywords for coordinate systems.
+    If `"Other"`, provide definition of the coordinate system in
+    `NIRSCoordinateSystemDescription`.
+  anyOf:
+  - $ref: _MEGCoordSys
+  - $ref: _EEGCoordSys
+  - $ref: _StandardTemplateCoordSys
+  - $ref: _StandardTemplateDeprecatedCoordSys
+NIRSCoordinateSystemDescription:
+  name: NIRSCoordinateSystemDescription
+  description: |
+    Free-form text description of the coordinate system.
+    May also include a link to a documentation page or paper describing the
+    system in greater detail.
+  type: string
+NIRSCoordinateUnits:
+  name: NIRSCoordinateUnits
+  description: |
+    Units of the coordinates of `NIRSCoordinateSystem`.
+  type: string
+  enum:
+  - m
+  - mm
+  - cm
+  - n/a
+NIRSCoordinateProcessingDescription:
+  name: NIRSCoordinateProcessingDescription
+  description: |
+    Has any post-processing (such as projection) been done on the optode
+    positions (for example, `"surface_projection"`, `"none"`).
```

Comment:
I would say rather than "none", this field should be left empty or set to `n/a`. Also, the example `surface_projection` is very sparse -- would it be possible to have it a little more descriptive?

Reply:
Changed `none` to `n/a`. 👍 

There was a lengthy conversation with the community about different projections. This was the example we were able to agree on.